frontend/src/views/PublicCanteenStatisticsPage/index.vue

```diff
@@ -0,0 +1,379 @@
+      <v-row>
+        <v-col cols="12" sm="6" md="5">
+          <v-card class="fill-height text-center pt-4 pb-2 px-3 d-flex flex-column" outlined>
+            <v-img max-width="30" contain src="/static/images/badges/appro.svg" class="mx-auto" alt=""></v-img>
+            <v-card-text class="grey--text text--darken-2 px-1">
+              <span class="text-h5 font-weight-black" style="line-height: inherit;">
+                {{ statistics.approPercent }} %
+              </span>
+              <span class="text-body-2">
+                ont réalisé la mesure
+                <span class="font-weight-black">« {{ approMeasure.shortTitle.toLowerCase() }} »</span>
+              </span>
+            </v-card-text>
+            <v-card-actions class="px-1">
+              <router-link :to="{ name: 'KeyMeasurePage', params: { id: approMeasure.id } }" class="text-body-2">
+                <!-- TODO: more specific link text? -->
+                La mesure
+              </router-link>
+            </v-card-actions>
+          </v-card>
+        </v-col>
+        <!-- TODO: this was copied from CanteenPublication. Consider making a component -->
+        <v-col cols="12" sm="6" md="3">
+          <v-card class="fill-height text-center py-4 d-flex flex-column justify-center" outlined>
+            <v-card-text>
+              <span class="text-h5 font-weight-black">{{ statistics.bioPercent }} %</span>
+              <span class="text-body-2">
+                bio moyen
+              </span>
+            </v-card-text>
+            <div class="mt-2">
+              <v-img
+                contain
+                src="/static/images/quality-labels/logo_bio_eurofeuille.png"
+                alt="Logo Agriculture Biologique"
+                title="Logo Agriculture Biologique"
+                max-height="35"
+              />
+            </div>
+          </v-card>
+        </v-col>
+        <v-col cols="12" sm="6" md="4">
+          <v-card class="fill-height text-center py-4 d-flex flex-column justify-center" outlined>
+            <v-card-text>
+              <span class="text-h5 font-weight-black">{{ statistics.sustainablePercent }} %</span>
+              <span class="text-body-2">
+                durables et de qualité moyen
+              </span>
+            </v-card-text>
+            <div class="d-flex mt-2 justify-center flex-wrap">
+              <v-img
+                contain
+                v-for="label in labels"
+                :key="label.title"
+                :src="`/static/images/quality-labels/${label.src}`"
+                :alt="label.title"
+                :title="label.title"
+                class="px-1"
+                max-height="40"
+                max-width="40"
+              />
+            </div>
+          </v-card>
+        </v-col>
+      </v-row>
+      <h3 class="text-h5 mt-10 mb-8">Ces cantines ont aussi réalisé les mesures suivantes en {{ year }}</h3>
```

Comment:
Dans l'esprit des autres commentaires sur les titres au dessus ;)

```suggestion
      <h3 class="text-h6 font-weight-black mt-10 mb-8">Ces cantines ont aussi réalisé les mesures suivantes en {{ year }}</h3>
```